docs/development/module/fees.md

```diff
@@ -0,0 +1,209 @@
+  - a configurable parameter for a runtime to describe how this multiplier can change. This is
+    expressed via
+    [`FeeMultiplierUpdate`](/rustdocs/master/srml_transaction_payment/trait.Trait.html#associatedtype.FeeMultiplierUpdate).
+
+`NextFeeMultiplier` has the type `Fixed64`, which can represent a fixed point number with a billion
+points accuracy. So, given the final fee formula above, the final version would be:
+
+```
+fee =
+  base_fee +
+  len(tx) * length_fee +
+  WeightToFee(weight)
+
+final_fee = fee * NextFeeMultiplier
+```
+
+Updating the `NextFeeMultiplier` has a similar manner as `WeightToFee`. The
+[`FeeMultiplierUpdate`](/rustdocs/master/srml_transaction_payment/trait.Trait.html#associatedtype.FeeMultiplierUpdate)
+associated type in `transaction-payment` is defined as a `Convert<Fixed64, Fixed64>`, which should
+be read as: it receives the previous multiplier and spits out the next one.
+
+The default update function is inspired by the Polkadot network and implements a targeted adjustment
+in which a target saturation level of block weight is defined. If the previous block is more
+saturated, then the fees are slightly increases. Similarly, if less transaction than the target are
+in the previous block, fees are decreased by a small amount. More information about this can be
+found in the [web3 research
+page](https://research.web3.foundation/en/latest/polkadot/Token%20Economics/#relay-chain-transaction-fees).
+
+## Substrate Weight System
+
+> This section assumes that you have already read the `Weight` section
+
+The entire substrate runtime module library is already annotated with a simple and fixed weight
+system. A user can decide to use the same system or implement a new one from scratch. The latter is
+outside the scope of this document and is explained int he dedicated [`Weight`]() conceptual
+document.
+
+### Using the Default Weight
+
+The default weight annotation is beyond _simple_. Substrate, by default, uses _fixed_ weights and
+the enum representing them is as follows:
+
+```rust
+pub enum SimpleDispatchInfo {
+    /// A normal dispatch with fixed weight.
+    FixedNormal(Weight),
+    /// A normal dispatch with the maximum weight.
+    MaxNormal,
+    /// A normal dispatch with no weight.
+    FreeNormal,
+    /// An operational dispatch with fixed weight.
+    FixedOperational(Weight),
+    /// An operational dispatch with the maximum weight.
+    MaxOperational,
+    /// An operational dispatch with no weight.
+    FreeOperational,
+}
+```
+
+This enum groups all dispatches into _normal_ and _operational_ (which makes the
+implementation of `ClassifyDispatch` pretty trivial) and gives them a fixed Weight. Fixed in this
+context means that the arguments of the dispatch do not play any role in the weight. Dispatches
+classified as _operational_ are exempt from paying both the `base_fee`, and `length_fee`.
+
+A simple example of using this enum in your runtime is:
+
+```rust
```

Comment:
I would prefer that we use subheadings and separate out these different kinds of fees.

Then it would show up in table of contents, and would allow direct linking.

Comments can become normal text at this point. I can make this change at a later point.

Reply:
I don't get the suggestion: make the code a subheading, or `A simple example of using this enum in your runtime is:` a new one? 

yeah I don't mind you or anyone applying changes. I did my best to put the content right and if it needs reshaping then it is fine.